readme.md

```diff
@@ -790,6 +790,7 @@
 - [Crypto Currency Tools & Algorithms](https://github.com/Zheaoli/awesome-coins#readme) - Digital currency where encryption is used to regulate the generation of units and verify transfers.
 - [Diversity](https://github.com/folkswhocode/awesome-diversity#readme) - Creating a more inclusive and diverse tech community.
 - [Open Source Supporters](https://github.com/zachflower/awesome-open-source-supporters#readme) - Companies that offer their tools and services for free to open source projects.
+- [Open Source Documentation](https://github.com/saintmalik/awesome-oss-docs#readme) - Open-source software documentation for people whole love contributing to through non coding.
```

Comment:
```suggestion
- [Open Source Documentation](https://github.com/saintmalik/awesome-oss-docs#readme) - Open-source software documentation for people whole love contributing to through non-coding.
```

Reply:
And multiple typos.

Reply:
The title here does not match the PR title or list title. It should all be consistent.